Starforge.Vanilla/Tools/EntityTool.cs

```diff
@@ -85,5 +99,22 @@ public class EntityTool : Tool {
             Hold = new Rectangle(MapEditor.Instance.State.TilePointer.X * 8, MapEditor.Instance.State.TilePointer.Y * 8, 8, 8);
             HeldEntity.SetArea(Hold);
         }
+
+        public override void RenderGUI() {
+            ImGui.Text("Entities");
+            ImGui.SetNextItemWidth(235f);
+
+            if (SelectedEntity == null)
+                SelectedEntity = EntityRegistry.EntityPlacements[0];
```

Comment:
Please either make this one line
```suggestion
            if (SelectedEntity == null) SelectedEntity = EntityRegistry.EntityPlacements[0];
```
or with brackets
```suggestion
            if (SelectedEntity == null) {
                SelectedEntity = EntityRegistry.EntityPlacements[0];
            }
```